src/VisualStudio/Core/Def/ManagedEditAndContinueService.vsdconfigxml

```diff
@@ -2,21 +2,37 @@
 <!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Configuration xmlns="http://schemas.microsoft.com/vstudio/vsdconfig/2008">
   <DefineGuid Name="ManagedEditAndContinueServiceId" Value="A96BBE03-0408-41E3-8613-6086FD494B43"/>
-  <DefineGuid Name="ManagedEditAndContinueMessageSourceId" Value="58CDF976-1923-48F7-8288-B4189F5700B1"/>
+  <DefineGuid Name="DebuggeeModuleMetadataProviderMessageSourceId" Value="58CDF976-1923-48F7-8288-B4189F5700B1"/>
+  <DefineGuid Name="StateChangeListenerMessageSourceId" Value="730432E7-1B68-4B3A-BD6A-BD4C13E0566B"/>
   <ManagedComponent
```

Comment:
❔ This file is new to me. Where can I get more information about it?

Reply:
This is Concord configuration file. Not sure if there is any doc on it.